support-frontend/assets/helpers/user/user.js

```diff
@@ -163,6 +166,9 @@ const init = (dispatch: Function, actions: UserSetStateActions = defaultUserActi
     });
   } else if (emailFromBrowser) {
     dispatch(setEmail(emailFromBrowser));
+    dispatch(setStateField(window.guardian.geoip.stateCode));
```

Comment:
Should this be `window.guardian.user.address4 || window.guardian.geoip.stateCode` in all of the places we are calling dispatch?

Reply:
emailFromBrowser does not guarantee that there's a `window.guardian.user` set of values. If that data's there then it'll be captured by `windowHasUser` condition above.

Reply:
Thanks